service/src/test/java/greencity/service/ubs/ValuesForUserTableServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package greencity.service.ubs;
+
+import greencity.ModelUtils;
+import greencity.constant.ErrorMessage;
+import greencity.entity.user.User;
+import greencity.entity.user.employee.Employee;
+import greencity.enums.SortingOrder;
+import greencity.filters.CustomerPage;
+import greencity.filters.UserFilterCriteria;
+import greencity.repository.EmployeeRepository;
+import greencity.repository.UserRepository;
+import greencity.repository.UserTableRepo;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.*;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class ValuesForUserTableServiceImplTest {
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Mock
+    private UserTableRepo userTableRepo;
+
+    @Mock
+    private EmployeeRepository employeeRepository;
+
+    @InjectMocks
+    private ValuesForUserTableServiceImpl valuesForUserTableService;
+
+    @Test
+    void checkGetAllFields() {
+        User user = User.builder()
+            .orders(new ArrayList<>())
+            .violations(5)
+            .currentPoints(8)
+            .build();
+        Pageable pageable = PageRequest.of(1, 1);
+        Employee employee = ModelUtils.getEmployee();
+        List<Long> tariffsInfo = List.of(1L, 2L, 3L);
+        when(employeeRepository.findByEmail(anyString())).thenReturn(Optional.of(employee));
+        when(employeeRepository.findTariffsInfoForEmployee(anyLong())).thenReturn(tariffsInfo);
+        when(userRepository.getAllUsersByTariffsInfoId(anyLong())).thenReturn(tariffsInfo);
+        when(userTableRepo.findAll(any(UserFilterCriteria.class), anyString(), any(SortingOrder.class),
+            any(CustomerPage.class), Mockito.<Long>anyList())).thenReturn(new PageImpl<>(List.of(user), pageable, 1L));
+        valuesForUserTableService.getAllFields(new CustomerPage(), "column", SortingOrder.ASC, new UserFilterCriteria(),
```

Comment:
please check if the result of `valuesForUserTableService.getAllFields` contains all expected values